sass/px-login.scss

```diff
@@ -39,33 +39,37 @@ $inuit-enable-btn--bare : true;
 @import "px-list-bare-design/_objects.list-bare.scss";
 @import "px-actionable-design/_objects.actionable.scss";
 
-.btn {
-    font-size: 1rem;
+// Component
+
+.login-button {
+  width: 180px;
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  align-items: center;
 }
 
-.px-login-button {
-    color: var(--px-login-color, $gray13);
+.login-button--user-icon {
+  min-width: 30px;
+  height: 30px;
```

Comment:
I would also use $inuit-base-spacing-unit here because you want it to be dependent on the font size